03_how_ln_works.asciidoc

```diff
@@ -609,159 +621,117 @@ These fees are paid to the miner who mines that particular block.
 
 ==== Offline vs Online, Asynchronous vs Synchronous
 
-When a Bitcoin user sends funds to a destination address he does not need to know anything about the recipient.
-The recipient could be logically speaking "online" (e.g. run a node) or could be "offline" (e.g. does not run a node, does not use any wallet currently). 
-The transaction is registered in the blockchain, and at any later point in time the recipient can have a look at it and see that has has received funds in the past. 
-In Bitcoin, the recipient can be "offline" and the interaction is asynchronous, sender and receiver do not need to synchronize in any way.
-
-In Lightning, this is different. 
-The node that represents the payee must be "online" during the transaction. 
-The payee must run a node or have someone that runs a node for him.  
-That node must be "online" at the time of the payment.
-To be precise, both nodes, the payer's and the payee's must be online at the time of payment.
-The payer and payee must cooperate, i.e. in some fashion one could say "synchronize" , during the payment. 
-If the payee's node is down the payer cannot make the payment.
-
-==== Satoshis vs. Milli-Satoshis
-On Bitcoin the smallest amount is a "satoshi" which cannot be divided any further.
-Lightning can be more flexible. 
-Lightning nodes can optionally be configured to work with "milli-satoshis". 
-So, within Lightning even smaller amounts can be exchanged. 
-When settled on-chain, it must be settled in "satoshis".
-
-==== Blockchain Space as Scarcity vs. Channel Capacity as Scarcity
-Bitcoin and Lightning have different scarce resources. 
-The scarce resource for operating on the Bitcoin protocol is the space in the blockchain. 
-The scarce resource for operating on Lightning is the channel capacity, or in other words the liquidity.
+When a Bitcoin user sends funds to a destination address he does not need to know anything about the recipient. The recipient could be might be offline or online, and no interaction between sender and recipient is needed. The interaction is between sender and the Bitcoin blockchain. Receiving bitcoin on the Bitcoin blockchain is a _passive_ and _asynchronous_ activity that does not require any interaction by the recipient, or for the recipient to be online at any time. Bitcoin addresses can even be generated offline and are never "registered" with the Bitcoin network. Only spending bitcoin requires interaction.
+
+In Lightning, the recipient must be "online" during the transaction.
+The recipient must run a node or have someone that runs a node on their behalf (a third-party custodian). To be precise, both nodes, the sender's and the recipient's must be online at the time of payment and must coordinate. Receiving a Lightning payment is an _active_ and _synchronous_ activity between sender and recipient, without the participation of most of the Lightning network or the Bitcoin network (except for the intermediary routing nodes, if any).
```

Comment:
this is actually not quite precise. theoretically a lightning  payment can be made if sender and receiver are never online at the same time (what is same time anyway c.f.: https://en.wikipedia.org/wiki/Spacetime ) but they need to be online several times during the expiry time of the invoice / timelock.